doc/modules/language-guide/pages/stablememory.adoc

```diff
@@ -0,0 +1,112 @@
+# The `StableMemory` library
```

Comment:
Shouldn't this still say `ExperimentalStableMemory`?

Reply:
```suggestion
# The `ExperimentalStableMemory` library
```